python/mlc_chat/op/gemm.py

```diff
@@ -0,0 +1,72 @@
+"""Operators enabled by external modules."""
+import operator
+from functools import reduce
+from typing import Optional
+
+from tvm.relax.frontend import nn
+from tvm.relax.frontend.nn import op
+
+
+def faster_transformer_dequantize_gemm(
+    x: nn.Tensor,
+    weight: nn.Tensor,
+    scale: nn.Tensor,
+    bias: Optional[nn.Tensor] = None,
+    group_size: Optional[int] = None,
+):
+    """
+    Faster Transformer dequantize gemm inference with CutlassFpAIntB
+
+    Parameters
+    ----------
+    x : nn.Tensor
+        The input tensor, with shape of [*m, k].
+
+    weight : nn.Tensor
+        The quantized weight data tensor, with shape of [k, n // num_elem_per_storage].
+
+    scale : nn.Tensor
+        The quantized weight scale tensor, with shape of [k // group_size, n].
+
+    bias : Optional[nn.Tensor]
+        The optional bias for matmul, with shape of [*_, k].
+
+    group_size : Optional[int]
+        The optional group size. If not set, then using k as group size.
+
+    Returns
+    ------
+    ret: nn.Tensor
+        The output tensor of deocde matmul.
```

Comment:
nit: the input shape/dtype look really neat to me, so do you want to add the output shape/dtype as well?